pkg/i18n/dutch.go

```diff
@@ -255,6 +255,7 @@ func dutchTranslationSet() TranslationSet {
 		LcStashStagedChanges:                "stash staged wijzigingen",
 		LcStashOptions:                      "Stash opties",
 		NotARepository:                      "Fout: moet in een git repository uitgevoerd worden",
+		DisplayConfirmationWindow:           "Bevestiging weergeven wanneer subproces wordt beëindigd",
```

Comment:
I'm not sure if this is translated but this translation is on point :)

Reply:
This was my first PR, so I wasn't really sure about the process for internationalization. :)